src/site/content/en/blog/web-otp/index.md

```diff
@@ -399,6 +397,17 @@ You may also fork it and create your version:
 
 ## FAQ
 
+### The dialog doesn't appear though I'm sending a properly formatted message. What's going wrong? {: #no-dialog}
```

Comment:
Think about the order that you have implied by this. You didn't give these instruction above. Now the reader who thinks they got everything has to figure out why their code doesn't work. Then they find this and thinks, "Why didn't they provide this information to begin with?"

Reply:
I wanted to keep this in FAQ but wanted to avoid duplicate notes earlier. Yes, I know it should be noted earlier as well. Do you think it's ok to write the same thing twice in this article?

Reply:
No. What I'm saying is, if the article covers this in a logical place to begin with, why would it need to be a frequently asked question?

Reply:
Because I'm assuming this will be asked frequently. It's easier for readers to look for FAQ than read from the beginning to find the information. I would keep it at FAQ.

Reply:
I'm OK with the current organization. Eiji has mentioned the topic in a contextually relevant place within the main flow of the document. If this content was only in the FAQ, and nowhere else, I would side with Joe. FAQs are a very common pattern and it's plausible that some people find them useful. The main thing is to ensure that we don't get lazy and only use FAQs to present information. See https://kayce.basqu.es/blog/FAQs/